internal/resources/fetcher.go

```diff
@@ -80,6 +120,10 @@ func (f *Fetcher) GetResourcesForPolicies(ctx context.Context, policies []polici
 	return auditableResources, nil
 }
 
+func (f *Fetcher) GetClusterWideResourcesForPolicies(ctx context.Context, policies []policiesv1.Policy) ([]AuditableResources, error) {
+	return f.getResources(ctx, policies, constants.NoNamespace)
```

Comment:
Thinking about what I've commented above, this is where I would move all the code